pkg/env/costmodelenv.go

```diff
@@ -99,6 +99,8 @@ const (
 	regionOverrideList = "REGION_OVERRIDE_LIST"
 )
 
+const DefaultConfigMountPath = "/var/configs"
```

Comment:
I'm not sure that this should be exported, because theoretically it should only be accessed via one of the `env.Get` function. Were you thinking along different lines?

Reply:
In the long term, I'd like that. But for now, we need to just export that precise string, otherwise we risk breaking behavior. And I think this is just as good, if not better than:
```
func GetDefaultConfigMountPath() string {
    return "/var/configs"
}
```

My thought is:  today, a LOT of places are hardcoded to the string `"/var/configs"` and if we make that `GetConfigPathWithDefault("/var/configs")` then there's a chance we change behavior on some clusters.

Is that a problem? Definitely. Am I in a position to solve it today? I think... no, unfortunately.

Reply:
Let's revisit later.